### PR TITLE
Omnibus config cleanup

### PR DESCRIPTION
**Pull request by TheDZhon**
This PR provides possible resolution for the #18 issue.

### CODE REVIEW

**Review by arwer13 — APPROVED** [review]

**Review by bulbozaur — APPROVED** [review]

**Review by kadmil — APPROVED** [review]
LGTM

Apart from that, we'll get different setups automagically depending on the network used while running test, and it's shouldn't be specified on the code level, right?

**Comment by TheDZhon** [comment]
@kadmil 
> LGTM
> 
> Apart from that, we'll get different setups automagically depending on the network used while running test, and it's shouldn't be specified on the code level, right?

Yep, the `conftest.py` also automatically grabs addresses to build contract test fixtures from the proper config depending on the provided network name

File: README.md [hunk 1]
```diff
@@ -1,41 +1,64 @@
...
 
-To run scripts on actually mainnet you need to add param `--network mainnet` to the end of the command and set the following env variables:
+To reveal a full test output pass the `-s` flag when running test scripts with `brownie test`
+#### Fork-mode development networks
 
+By default, you should start testing process in a network fork mode (don't forget to review brownie fork params). The Mainnet-fork mode exists in brownie by default. To add the Görli Testnet fork mode please run once the following command (to setup and save new network):
 ```bash
-export DEPLOYER=<brownie_wallet_name>
-export WEB3_INFURA_PROJECT_ID=<infura_key>
+brownie networks add "Development" goerli-fork host=http://127.0.0.1 cmd=ganache-cli port=8545 gas_limit=12000000 fork=https://goerli.infura.io/v3/${WEB3_INFURA_PROJECT_ID} chain_id=5 mnemonic=brownie accounts=10
```
Comment (folkyatina) [inline README.md:38]: Can it be added directly to brownie-config.yml?

Comment (TheDZhon) [reply README.md:38]: Nope, I wish it was so, but the `goerli-fork` testnet needs to be added explicitly first into the brownie networks list

**Pull request merged**